xrviz/control.py

```diff
@@ -11,24 +11,29 @@
 
 class Control(SigSlot):
     """
-    This section allows the user to control the other subsections,
-    such as displayer, fields.
+    This section arranges all the panes in the form of tabs,
+    hence providing the controls to inteact with all the
+    panes.
 
     Parameters
     ----------
-    data: `xarray` instance: `DataSet` or `DataArray`
-           datset is used to initialize.
+    data: `xr.core.dataarray.DataWithCoords`
+        Data is required for initialization.
 
     Attributes
     ----------
-    panel: Displays the generated template.
-    displayer: Provides access to `Display` sub-section.
-    describer: Provides access to `Describe` sub-section.
-    fields: Provides access to `Fields` sub-section.
-    kwargs: Provides access to kwargs selected in different subsections.
+    1. ``panel``: Displays all the panes arraged in form of tabs.
+    2. ``displayer``: Provides access to `Display` sub-section of `Variables` pane.
+    3. ``describer``: Provides access to `Describe` sub-section of `Variables` pane.
+    4. ``coord_setter``: Provides access to the `Set Coords` pane.
+    5. ``fields``: Provides access to `Axes` pane.
+    6. ``style``: Provides access to `Style` pane.
+    7. ``projection``: Provides access to `Projection` pane (if present).
+    8. ``kwargs``: Provides access to values selected in different panes.
```

Comment:
None of these "provides access" statements are very useful. An attribute always provides access to something. Be very specific, as a scientist would. This last one, for instance:
"a dictionary gathered from the widgets of the input Panes, of a form which can be passed to the plotting function as kwargs"